test/model/adapters/sql_adapter_test.rb

```diff
@@ -1175,5 +1175,25 @@ class TestDeviceRepository
         end
       end
     end
+
+    describe 'execute' do
+      before do
+        @adapter.create(collection, user1)
+        @adapter.create(collection, user2)
+        @adapter.create(collection, TestUser.new(name: 'S'))
+      end
+
+      it 'returns the ResultSet from the executes sql' do
+        sql = "select * from users"
+        result = @adapter.execute(sql)
+        result.class.name.must_equal "SQLite3::ResultSet"
+        result.count.must_equal 3
+      end
+
+      it 'raises an exception when an invalid sql is provided' do
+        sql = "select foo from bar"
+        -> { @adapter.execute(sql) }.must_raise Sequel::DatabaseError
```

Comment:
@peterberkenbosch With #180 we have introduced `Lotus::Model::InvalidQueryError`. The long term goal is to let Lotus::Model to not raise adapters low level exceptions. Can you please catch `Sequel::DatabaseError` in the adapter and re-raise it? Thank you! :confetti_ball: